test/e2e/specs/editor/various/multi-block-selection.spec.js

```diff
@@ -752,16 +752,13 @@ test.describe( 'Multi-block selection (@firefox, @webkit)', () => {
 				name: 'Block: Paragraph',
 			} )
 			.filter( { hasText: '1' } );
+		// For some reason in Chrome it requires two clicks, even though it
+		// doesn't when testing manually.
 		await paragraph1.click( {
 			position: { x: -1, y: 0 },
 			// Use force since it's outside the bounding box of the element.
 			force: true,
 		} );
-
-		await expect
-			.poll( multiBlockSelectionUtils.getSelectedFlatIndices )
-			.toEqual( [ 1 ] );
```

Comment:
So this is a weird failure: we're actually also expecting this test to clear selection in trunk (it even says it in the test title), but for some reason after the first click we check if one block is selected, and on check if selection is cleared after the second click. This caused the webkit test to fail, it now always clears the selection. When I tested Chrome manually, it did the same, but for some reason with Playwright it's still relying on two clicks. 

I've also included Firefox now which seems to pass after these changes.